lib/viewcomponents/node.jsx

```diff
@@ -64,20 +66,20 @@ var Node = React.createClass({
   },
 
   _onNodeBodyPseudoDrag: function(event) {
-      var oldPos = this.props.model.getPosition();
+      var oldPos = this.props.model.get('position');
       
       var newX = oldPos.x + event.movementX;
       var newY = oldPos.y + event.movementY;
 
       // Change the position of the node in the model.
-      this.props.globalModel.moveNode(this.props.model.getId(), newX, newY);
+      this.props.model.setPosition(newX, newY);
```

Comment:
Could do that too. I just created a convenience method. Would you prefer removing the convenience method?